iothub/device/src/DeviceAuthenticationWithX509Certificate.cs

```diff
@@ -45,7 +45,6 @@ public string DeviceId
         /// <summary>
         /// Full chain of certificates from the one used to sign the device certificate to the one uploaded to the service. Private keys are not required for these certificates.
         /// This is only supported on AMQP_Tcp_Only and Mqtt_Tcp_Only
-        /// See <see href="TODO (Sample ref) - Sindhu"/>
```

Comment:
Realized samples depend on packages and not project ref. So I cannot send out a PR in samples repo till we release a new package.